client/reducers/reducer.js

```diff
@@ -1,7 +1,8 @@
-// import { immutable, fromJS, toJS} from 'immutable'
+// import { immutable, fromJS, toJS} from a.immutable
```

Comment:
Would you please delete this comment and uninstall immutable from package.json if it is still there